src/main/java/com/example/tweet/entities/UserForm.java

```diff
@@ -0,0 +1,22 @@
+package com.example.tweet.entities;
+
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+
+@Data
+public class UserForm  {
+
+    @NotEmpty(message = "名前が未入力です。")
+    @Size(max=20 ,message = "名前は２０文字以内で入力してください。")
```

Comment:
```suggestion
    @Size(max=20 ,message = "名前は20文字以内で入力してください。")
```
半角、全角はメッセージ内で統一お願いします。